src/sbom.rs

```diff
@@ -79,13 +80,18 @@ async fn get_sbom(scan_target: OsString, sbom_path: PathBuf) -> Result<Value> {
             .arg("inspect")
             .arg(scan_target)
             .arg("--format")
-            .arg(format!("{{{{ json (index .SBOM \"{arch}\").SPDX }}}}"));
+            .arg("{{ json .SBOM }}");
 
         let output = command.output().await?;
+        let output: Value = serde_json::from_slice(&output.stdout)?;
 
-        let parsed_output: Value = serde_json::from_slice(&output.stdout)?;
+        let parsed_output = match output.get(arch) {
+            Some(v) => v.get("SPDX"),
+            None => output.get("SPDX"),
+        }
+        .context("Image does not have compatible sbom attestation")?;
```

Comment:
This will fall back to generating the sbom, right?

Reply:
yes, if generating sboms is enabled and there is neither a cached sbom, nor an sbom attestation, it'll continue on to calling syft.